### PR TITLE
Redis: adding annotation and service type to redis setup

### PR DESCRIPTION
**Pull request by ghost**
Adding annotations and service type for redis chart.

### CODE REVIEW

File: stable/redis/values.yaml [hunk 1]
```diff
@@ -87,3 +87,7 @@ networkPolicy:
   ##
   allowExternal: true
 
+service:
+  type: ClusterIP
+  annotations: {}
+  loadBalancerIP:
```
Comment (mattfarina) [inline stable/redis/values.yaml]: According to the linter:

>  93:18     error    no new line character at the end of file  (new-line-at-end-of-file)

Please add a new line at the end of the file.

Comment (ghost) [reply stable/redis/values.yaml]: Thanks @mattfarina. Done.

**Comment by unguiculus** [comment]
Please add documentation to the readme.

**Pull request merged**